package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bigcommerce/stencil-cli",
-  "version": "1.14.3",
+  "version": "1.15.0",
```

Comment:
I thought we used `npm version` & `npm publish` to update versions.

Reply:
We could, but this basically does the same thing.